github/plugin.py

```diff
@@ -615,6 +641,32 @@ def modify_repo(self, repo, org, **kwargs):
                 .format(response.status_code)
             raise requests.exceptions.RequestException(message)
 
+    def get_repo_pull_requests_http(self, repo, org, state, **kwargs):
+        """
+        List pull requests associated with a repo.
+
+        :param repo:
+        :param org:
+        :param state:
+        :param kwargs:
+        :return:
+        """
+        headers = {
+            'Authorization': 'token {}'.format(self.token),
+            'Accept': GITHUB_VERSION
+        }
+
+        api_part = 'repos/{}/{}/pulls?state={}'.format(org, repo, state)
```

Comment:
I think for a first version we can go with the default limit, and since the practical usage will be usually just to quick check the most recent PRs (open, closed).

For advanced usage, the user might be interested to perform his actions on the Github search engine directly ...